dali/operators/generic/pad.cc

```diff
@@ -62,8 +61,9 @@ Samples are padded in the first axis to a minimum extent of 7.
 
 - `1-D` samples, `fill_value` = -1, `axes` = (0,), `align` = (4,)
 
-Samples are padded in the first axis to match the extent of the largest sample and the alignment requirements
-(largest extent is 5 but it is extended to 8 to satisfy the alignment requirements).
+The samples are padded in the first axis to match the extent of the largest sample and the
+alignment requirements. The largest extent is 5, but it is extended to 8 to satisfy the
+alignment requirements.
```

Comment:
```suggestion
alignment requirements. The output extent is 8 which is a result of rounding the maximum extent (5) to a
multiple of alignment (4).
```
I think that "extent was extended" sounds bad...

Reply:
Done